exercises/practice/twelve-days/.meta/example.cpp

```diff
@@ -0,0 +1,51 @@
+#include "twelve_days.h"
+#include <array>
+#include <string>
+
+namespace twelve_days {
+
+std::string verse(int day) {
+    const std::array<std::string, 13> ordinals = {
+        "", "first", "second", "third", "fourth", "fifth", "sixth",
+        "seventh", "eighth", "ninth", "tenth", "eleventh", "twelfth"
+    };
+    
+    const std::array<std::string, 13> gifts = {
+        "", "a Partridge in a Pear Tree", "two Turtle Doves", "three French Hens",
+        "four Calling Birds", "five Gold Rings", "six Geese-a-Laying",
+        "seven Swans-a-Swimming", "eight Maids-a-Milking", "nine Ladies Dancing",
+        "ten Lords-a-Leaping", "eleven Pipers Piping", "twelve Drummers Drumming"
+    };
+    
+    auto result = std::string{"On the "} + ordinals[day] + " day of Christmas my true love gave to me: ";
+    
+    for (int i = day; i >= 1; --i) {
+        if (i == day) {
+            result += gifts[i];
+        } else if (i == 1) {
+            result += std::string{", and "} + gifts[i];
+        } else {
+            result += std::string{", "} + gifts[i];
+        }
+    }
+    
+    result += ".";
+    return result;
+}
+
+std::string lyrics(int start_day, int end_day) {
+    std::string result = "";
+    for (int i = start_day; i <= end_day; ++i) {
+        if (i > start_day) {
+            result += "\n\n";
```

Comment:
The tests look like you expect a newline after each verse. Then adding `\n\n` between two verses would make that triple newline characters between verses.